src/06_tuples.py

```diff
@@ -35,10 +35,18 @@ def dist(a, b):
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(t):
+    for val in t:
+        print(val)
+
 
 t = (1, 2, 5, 7, 99)
 print_tuple(t)  # Prints 1 2 5 7 99, one per line
 
+def print_tuple(t):
+    for val in t,:
+        print(val)
 # Declare a tuple of 1 element then print it
```

Comment:
You yet to answer 
What needs to be added to make this work?

Reply:
I believe i added the comma, though i didn't state it explicitly